app/models/invoice.rb

```diff
@@ -8,5 +8,6 @@ class Invoice < ApplicationRecord
   belongs_to :customer
   has_many :invoice_items
   has_many :items, through: :invoice_items
+  has_many :merchants, through: :items
```

Comment:
glad you added this relationship, in my head I thought we might need it but couldn't think of a use case